app/views/shared/_invite_users_modal.html.erb

```diff
@@ -99,7 +99,7 @@ invite_to_team = type.in?(%w(invite_to_team invite_to_team_with_role))
               </div>
             <% end %>
           <% end %>
-          <% if ENV['ENABLE_RECAPTCHA'] %>
+          <% if ENV['ENABLE_RECAPTCHA'] == 'true' %>
```

Comment:
Few tasks ago I figure out that we are using `secrets`. Should we go with `Rails.secrets...` instiead of checking environment variables directly?

Reply:
Wouldn't this add the extra step of defining everything in `secrets.yml` or do you believe all the variables should be defined in `secrets.yml`?